web/pgadmin/browser/server_groups/servers/databases/schemas/tables/columns/tests/utils.py

```
@@ -193,7 +193,7 @@ def verify_column(server, db_name, col_name):
                                              server['port'],
                                              server['sslmode'])
         pg_cursor = connection.cursor()
-        pg_cursor.execute("select * from pg_attribute where attname='%s'" %
+        pg_cursor.execute("select * from pg_catalog.pg_attribute where attname='%s'" %
                           col_name)
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)